tfhe/benches/core_crypto/pbs_bench.rs

```diff
@@ -850,6 +850,287 @@ mod cuda {
+                underlying_container,
+                params.lwe_dimension.unwrap().to_lwe_size(),
+                params.ciphertext_modulus.unwrap(),
+            );
+
+            let output_lwe_list = LweCiphertextList::new(
+                Scalar::ZERO,
+                big_lwe_dimension.to_lwe_size(),
+                LweCiphertextCount(NUM_CTS),
+                params.ciphertext_modulus.unwrap(),
+            );
+            let lwe_ciphertext_in_gpu =
+                CudaLweCiphertextList::from_lwe_ciphertext_list(&input_lwe_list, &stream);
+
+            let accumulator = GlweCiphertext::new(
+                Scalar::ZERO,
+                params.glwe_dimension.unwrap().to_glwe_size(),
+                params.polynomial_size.unwrap(),
+                tfhe::core_crypto::prelude::CiphertextModulus::new_native(),
+            );
+            let accumulator_gpu =
+                CudaGlweCiphertextList::from_glwe_ciphertext(&accumulator, &stream);
+
+            let mut out_pbs_ct_gpu =
+                CudaLweCiphertextList::from_lwe_ciphertext_list(&output_lwe_list, &stream);
+            let mut h_indexes: [Scalar; NUM_CTS] = [Scalar::ZERO; NUM_CTS];
+            let mut d_lut_indexes = unsafe { CudaVec::<Scalar>::new_async(NUM_CTS, &stream) };
+            unsafe {
+                d_lut_indexes.copy_from_cpu_async(h_indexes.as_ref(), &stream);
+            }
+            stream.synchronize();
+            for (i, index) in h_indexes.iter_mut().enumerate() {
+                *index = Scalar::cast_from(i);
+            }
+            stream.synchronize();
+            let mut d_input_indexes = unsafe { CudaVec::<Scalar>::new_async(NUM_CTS, &stream) };
+            let mut d_output_indexes = unsafe { CudaVec::<Scalar>::new_async(NUM_CTS, &stream) };
+            unsafe {
+                d_input_indexes.copy_from_cpu_async(h_indexes.as_ref(), &stream);
+                d_output_indexes.copy_from_cpu_async(h_indexes.as_ref(), &stream);
+            }
+            stream.synchronize();
+
+            let id = format!("{bench_name}::{name}::{NUM_CTS}chunk");
+            bench_group.bench_function(&id, |b| {
+                b.iter(|| {
+                    cuda_programmable_bootstrap_lwe_ciphertext(
+                        &lwe_ciphertext_in_gpu,
+                        &mut out_pbs_ct_gpu,
+                        &accumulator_gpu,
+                        &d_lut_indexes,
+                        &d_output_indexes,
+                        &d_input_indexes,
+                        LweCiphertextCount(NUM_CTS),
+                        &bsk_gpu,
+                        &stream,
+                    );
+                    black_box(&mut out_pbs_ct_gpu);
+                })
+            });
+
+            let bit_size = params.message_modulus.unwrap().ilog2();
+            write_to_json(
+                &id,
+                *params,
+                name,
+                "pbs_throughput",
```

Comment:
Just `"pbs"` the throughput notion is a benchmark detail. But the operation is ultimately just a PBS.